### DISCUSSION TITLE
How are you updating dependencies of this project like - uosc and thumbfast

### DISCUSSION THREAD

**Discussion by KAGEYAM4** [hooke007/mpv_PlayKit#377] [z_ENG_discuss]
I also want to build personal mpv config but at very small scale for personal use. There are some plugins that i want to use, like uosc and thumbfast.

But the problem is how can i update them.

What methods are you using? Like in git? 

I saw this commit [Script: uosc sync to 5.0.0+](https://github.com/hooke007/MPV_lazy/commit/186baff7f34ec1d7e0d821b5ec71bffa2efe5f22), how can i do this similary in my project?

**Comment by hooke007** [comment]
I've made tons of changes to my fork compared to the original version. So I had to track every commits upstream and decide if it should be merged into my fork.
The commit you refer looks like only one commit. Actually it was squashed from many commit in my local machine over the past months.

**Reply by KAGEYAM4** [reply]
> So I had to track every commits upstream and decide if it should be merged into my fork.

How are you doing this part? I want to do the same.

Are you using git-subtree or git-submodules?

**Reply by KAGEYAM4** [reply]
Which git command you use to pull changes from [External Scripts](https://github.com/hooke007/MPV_lazy/wiki/2_portable_config#%E5%A4%96%E7%BD%AE%E8%84%9A%E6%9C%AC%E8%B7%AF%E5%BE%84-) and how do you selectively merge them in you main-branch

**Reply by hooke007** [reply]
Well, no git commands, it is almost done manually by my self.

**Reply by KAGEYAM4** [reply]
ok. i thought there was some easy automation as i was looking for one. guess i will keep looking.

Thank you for your prompt response.

**Reply by hooke007** [reply]
> there was some easy automation as i was looking for one. 

Me too :-)
